backend/business-services/egov-hrms/src/main/resources/config/hrm-employee-update-persister.yml

```
@@ -0,0 +1,265 @@
+    name: hrms
+    description: Persists employee details in the table
+    fromTopic: update-hrms-employee
+    isTransaction: true
+    queryMaps:
+
+    - query: DELETE from eg_hrms_employee WHERE uuid=? on
+
+      basePath: Employees.*
+      jsonMaps:
+
+        -jsonPath:  $.Employees.*.uuid
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)